styleguide/components/31-molecules/teasers/teaser-article/teaser-article.config.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
```

Comment:
Gingen we in stijlgids niet gaan voor Engelstalige voorbeeld content?

Reply:
👍

Reply:
I'm sorry, didn't quite catch that. Could you use English please?